packages/core/src/modules/dids/methods/sov/IndySdkSovDidRegistrar.ts

```diff
@@ -102,7 +98,8 @@ export class SovDidRegistrar implements DidRegistrar {
       // Build did document.
       const didDocument = didDocumentBuilder.build()
 
-      const didIndyNamespace = this.indyPoolService.ledgerWritePool.config.indyNamespace
+      const pool = this.indyPoolService.getPoolForNamespace(options.indyNamespace)
```

Comment:
We can pass this pool to the `registerPublicDid` and `setEndpointsForDid` methods